wingtips-core/src/test/java/com/nike/wingtips/http/HttpRequestTracingUtilsTest.java

```diff
@@ -260,6 +260,51 @@ public void fromRequestWithHeaders_sets_sampleable_to_attribute_value_if_sampled
         assertThat(newSpan.isSampleable()).isFalse();
     }
 
+    @DataProvider(value = {
```

Comment:
haha how could I have missed that :)

On Sat, Aug 20, 2016 at 9:52 AM, Nic Munroe <notifications@github.com>
wrote:

> In wingtips-core/src/test/java/com/nike/wingtips/http/
> HttpRequestTracingUtilsTest.java
> <https://github.com/Nike-Inc/wingtips/pull/16#discussion_r75570549>:
>
> > @@ -260,6 +260,51 @@ public void fromRequestWithHeaders_sets_sampleable_to_attribute_value_if_sampled
> >          assertThat(newSpan.isSampleable()).isFalse();
> >      }
> >
> > +    @DataProvider(value = {
>
> There's an already-existing test that verifies missing/empty sampled
> header coerces to true: fromRequestWithHeaders_sets_
> sampleable_to_true_if_sampled_header_is_null_or_missing_
> from_both_headers_and_attributes.
>
> There's a separate mechanism for determining whether new traces are
> sampled: RootSpanSamplingStrategy.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nike-Inc/wingtips/pull/16/files/f037fd2cb5d00e196447cbaa83a39de830394e7b#r75570549>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD612lNDUkV8E3uWSG7OylxgCSdno1Tks5qhl3XgaJpZM4Jo1BS>
> .
>